webapp/accounts/templates/base.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
```

Comment:
We already have a base.html template in `master` under the global app.
We do not need to duplicate it here; please delete this and adjust the code accordingly.